housekeeper/cli/add.py

```diff
@@ -36,16 +36,20 @@ def bundle(context, name):
 @add.command('file')
 @click.option('-t', '--tag', 'tags', multiple=True, help='tag to associate the file by')
 @click.option('-a', '--archive', is_flag=True, help='mark file to be archived')
+@click.option('-v', '--version', help='the creation datetime of the version')
```

Comment:
If possible it would be nice to add the expected format (e.g. [YYMMDD] or how it's defined)